### DISCUSSION TITLE
Grid Rowspan

### DISCUSSION THREAD

**Discussion by juangdnar** [filamentphp/filament#4710] [Help] [answered]
Is it possible to make more complex groupings of grids without using ViewFields? I would like to be able to have two columns, and in the second a rowspan of 3 depending on the first row, as shown in the image:

![image](https://user-images.githubusercontent.com/115037170/197831124-8ff0a544-b1db-4aed-9559-b7f4b7b9c00b.png)

**Comment by KaanTanis** [comment, marked as answer]
```php
Card::make()
    ->schema([
        Group::make()
            ->schema([
                TextInput::make('title'),
                TextInput::make('description'),
                TextInput::make('test'),
            ]),

        Group::make()
            ->schema([
                RichEditor::make('content'),
            ]),
    ])->columns(2),

Card::make()
    ->schema([
        Group::make()
            ->schema([
                TextInput::make('tags'),
            ])->columnSpan(2),
    ])
```

![image](https://user-images.githubusercontent.com/64961969/198274982-fbe213e9-f5b6-48da-a893-c67b3067496c.png)

**Comment by juangdnar** [comment]
Thanks a lot!!!
